src/vtquery.cpp

```diff
@@ -107,7 +117,8 @@ struct QueryData {
           radius(0.0),
           num_results(5),
           dedupe(true),
-          geometry_filter_type(GeomType::all) {
+          geometry_filter_type(GeomType::all),
+          filters() {
```

Comment:
This part doesn't actually work yet, need to figure out how to pass stuff into C++ land from Node.